contracts/mocks/VoucherV2Mock.sol

```diff
@@ -31,7 +32,7 @@ contract VoucherV2Mock is OwnableUpgradeable {
      * Initialize new implementation contract.
      * @param newStateVariable test variable.
      */
-    function initializeV2(uint256 newStateVariable) external reinitializer(2) {
+    function initializeV2(uint256 newStateVariable) external {
```

Comment:
I think it's there as documentation/to not forget.

Reply:
I don't understand sorry

Reply:
I mean is there a reason to remove it?

Reply:
Actually it's a mistake thanks for the review I put it back